demo/index.html

```diff
@@ -17,22 +17,29 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
 
     <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
+
+    <link href="../../paper-styles/paper-styles.html" rel="import">
+    <link href="../../paper-styles/demo-pages.html" rel="import">
     <link rel="import" href="../iron-media-query.html">
```

Comment:
`rel=import` should go at the beginning (i think i missed this a bunch of times too)